### PR TITLE
fix: track desktop events

### PR DESCRIPTION
**Pull request by 2fd**

### CODE REVIEW

**Review by menduz — APPROVED** [review]
looks good

File: src/integration/kernel.ts [hunk 1]
```diff
@@ -20,8 +21,12 @@ export const initializeKernel = callOnce(() => {
   fadeoutElement('root-loading')
 })
 
+export function configureKernel(store: Store<StoreType>) {
+  store.subscribe(() => hideRoot(store.getState()))
+}
+
 let ROOT_HIDDEN = false
-export const hideRoot = (state: StoreType) => {
+function hideRoot (state: StoreType) {
```
Comment (nachomazzara) [inline src/integration/kernel.ts:29]: It is not being used anywhere, right?

**Review by nachomazzara — APPROVED** [review]
LGTM

File: src/integration/kernel.ts (hunk 1 above)
Comment (2fd) [reply src/integration/kernel.ts:29]: just in this file

**Review by kuruk-mm — APPROVED** [review]
Awesome!

**Review by nachomazzara — APPROVED** [review]

**Pull request merged**